ansible_collections/arista/avd/roles/eos_designs/templates/network_services/router-bgp-vrfs.j2

```diff
@@ -56,7 +56,10 @@
                                                          tenant.enable_mlag_ibgp_peering_vrfs,
                                                          true) %}
 {%                         if configure_mlag_ibgp_peering %}
-        {{ switch.mlag_peer_l3_ip | arista.avd.default(switch.mlag_peer_ip) }}:
+{%                             if vrf.mlag_ibgp_peering_ipv4_pool is arista.avd.defined %}
+{%                                 set mlag_ibgp_peering_peer_ip = vrf.mlag_ibgp_peering_ipv4_pool | ansible.netcommon.ipaddr('subnet') | ansible.netcommon.ipmath(switch.mlag_peer_id - 1) %}
```

Comment:
Any reason for which the formula is not the same as in the other mlag_ip templates:
https://github.com/aristanetworks/ansible-avd/blob/707ca25feb8ee6e5a5f2d87c1eb6bb71dd10e0f6/ansible_collections/arista/avd/roles/eos_designs/templates/ip_addressing/mlag-l3-ip-primary.j2#L1
?

And another question - would it be possible to compute this as a fact maybe?

Reply:
This is set on Tenant networks for every VRF individually based on switch id. So I don't think of it as a switch fact. I can adjust the code to make it work like a switch fact but that wasn't intended.

Reply:
With respect to the formula, I derived it using both id and peer_id, which I need to change. That's why the formula is different.

Reply:
Using the same formula.